recipes/libassert/all/conandata.yml

```diff
@@ -0,0 +1,19 @@
+sources:
+  # Newer versions at the top
+  "1.1":
+    url:
+      - "https://github.com/jeremy-rifkin/libassert/archive/refs/tags/v1.1.tar.gz"
+    sha256: "01e1b99bf4dacd816fceeb5cc7efdc8875ed21295a62396584a0e7649b944c67"
+  "1.0":
+    url:
+      - "https://github.com/jeremy-rifkin/libassert/archive/refs/tags/v1.0.tar.gz"
+    sha256: "e1bb3b50767994ca4d0f60b7977b279cf32b8569ff92c5830e7a1de567b82fd5"
+
+patches:
+  "1.1":
+    - patch_file: "patches/1.1/0001-fix-max_line_number_width-miscalculation.patch"
```

Comment:
Can you please add `patch_type` fields? It helps us a lot when reviewing patches to know their pourpose and origin!